### DISCUSSION TITLE
Can the app hidden as a icon in the bottom right corner when be minimized ?

### DISCUSSION THREAD

**Discussion by GarenLancaster** [keepassxreboot/keepassxc#9401] [Ideas]
I use this app everyday and it is really helpful. But it take too much space in the taskbar, beacuse i use it in browser mostly don't need to open it frequently.If it can be minimized to a icon in the bottom right that would be really great.

**Comment by droidmonkey** [comment]
Go to application settings and scroll to the bottom

**Reply by GarenLancaster** [reply]
Thank you, this is really helpful(≧∇≦)ﾉ
